docker-compose.yml

```diff
@@ -419,6 +421,85 @@ services:
     ports:
       - "33105:3105"
 
+  manuals-publisher: &manuals-publisher
+    build: apps/manuals-publisher
+    depends_on:
+      - publishing-api
+      - mongo
+      - rummager
+      - static
```

Comment:
looking in the repo I wonder if the dependency on static is out of date: https://github.com/alphagov/manuals-publisher/search?utf8=%E2%9C%93&q=static&type=